src/Tests/dotnet-new.Tests/PostActionTests.cs

```diff
@@ -872,5 +872,198 @@ public void ItCanCreateTemplate_WithRestore()
                 .And.HaveStdOutContaining("Determining projects to restore...")
                 .And.HaveStdOutContaining("Restore succeeded.");
         }
+
+        [Fact]
+        public void AddJsonProperty_Basic()
+        {
+            const string templateLocation = "PostActions/AddJsonProperty/Basic";
+            const string templateName = "TestAssets.PostActions.AddJsonProperty.Basic";
+
+            string home = CreateTemporaryFolder(folderName: "Home");
+            string workingDirectory = CreateTemporaryFolder();
+            InstallTestTemplate(templateLocation, _log, home, workingDirectory);
+
+            new DotnetNewCommand(_log, templateName)
```

Comment:
Applies to other tests as well.

I would recommend to use snapshot test to assert the whole output as here: https://github.com/dotnet/sdk/blob/ec94dfe5a77140541bb1b87165de4372e64be223/src/Tests/dotnet-new.Tests/PostActionTests.Approval.cs#L32-L42

It is possible to "scrub" the output that may change.
If you'd like to assert both stdout and strerr you can use `FormatOutputStreams` extension: https://github.com/dotnet/sdk/blob/ec94dfe5a77140541bb1b87165de4372e64be223/src/Tests/dotnet-new.Tests/Extensions.cs#L11

Reply:
TBH, I don't really see an advantage of this approach.  It's somewhat bloated and it is not immediately visible what you is being tested imho.

Reply:
it's okay not to use it - we found it useful as it blocks unexpected output if the feature is broken in a  way.

Reply:
Then I would opt to not use it.  It is then also consistent with the other integration-tests for other Post Actions that are in the same file.